addons/website_sale/models/product_public_category.py

```diff
@@ -90,6 +90,19 @@ def _default_sequence(self):
         help="Align the category content on the shop page. Corresponds to the 'Center Content' editor option."
     )
 
+    not_in_shop = fields.Boolean(
```

Comment:
Move this above `website_description` field 🤔 also there are so many spaces b/w fields

I guess here we tried to group fields 
- First common fields
- Then Parent category related fields
- Then Product related
- Then Website related but little mess up

May be we should move this field in first section. WDYT?